nixos/modules/services/hardware/lirc.nix

```diff
@@ -0,0 +1,86 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lirc;
+in {
+
+  ###### interface
+
+  options = {
+    services.lirc = {
+
+      enable = mkEnableOption "LIRC daemon";
+
+      options = mkOption {
+        type = types.lines;
+        example = ''
+          [lircd]
+          nodaemon = False
+        '';
+        description = "LIRC default options (lirc_options.conf)";
+      };
+
+      config = mkOption {
+        type = types.lines;
+        description = "Configuration to load.";
+      };
+
+      extraArguments = mkOption {
+        type = types.str;
+        default = "";
```

Comment:
Make this a list, like this: https://github.com/NixOS/nixpkgs/blob/898aedcc042f929ecc35ce73c2bba81e63077137/nixos/modules/services/misc/zookeeper.nix#L94-L99